### ISSUE TITLE
getDistinctAttribute returning value with double comma

### ISSUE THREAD

**Issue by ppamorim** [meilisearch/meilisearch-swift#56]
## Summary

When calling the function `getDistinctAttribute` the result returns an invalid string that contains a double quote, like this:

```
"product_id"
```

Note the additional comma, when represented by Swift it prints as `""product_id""`. The value it should return is `product_id` instead.

## Origin

The problem comes from this line, Meilisearch returns `"product_id"` and Swift understand this as the full string.

https://github.com/meilisearch/meilisearch-swift/blob/9a2bf4caeffe160053f7e2892277427dcb04befe/Sources/MeiliSearch/Settings.swift#L418-L419

## Solution

I can't find any ideal way to parse this value, the only way I found, bit smelly, is to drop the first and last characters of the string and accept it as is.

## Help

@curquiza @qdequele @bidoubiwa Any idea of what can be done? Is my solution good enough (I don't like it)?

**Comment by qdequele** [comment]
Why did you not use :

**Comment by ppamorim** [comment]
@qdequele The struct `Update` requires a key-value structure with the only key `updateId`.

What I didn't know is that `String` complies with the `Decodable` protocol, so be code below is valid:

```swift
let distinctAttribute: String? = try? Constants.customJSONDecoder.decode(String.self, from: data)
```

I will close this issue when the PR with the correction gets merged.

**Comment by ppamorim** [comment]
Fixed in the #54.
